pkg/webhookadmission/utils.go

```diff
@@ -205,15 +226,23 @@ func createWebhookV1(caBundle []byte, ns string) error {
 		Webhooks: []admissionv1.MutatingWebhook{webhook},
 	}
 
-	// recreate webhook
-	err := admissionregistration.Instance().DeleteMutatingWebhookConfiguration(storkAdmissionController)
-	if err != nil && !k8serr.IsNotFound(err) {
+	// UpdateOrCreate v1 webhook
+	resp, err := admissionregistration.Instance().GetMutatingWebhookConfiguration(storkAdmissionController)
+	if err != nil {
+		if k8serr.IsNotFound(err) {
+			_, err = admissionregistration.Instance().CreateMutatingWebhookConfiguration(req)
+			if err != nil {
+				log.Errorf("Unable to create webhook configuration: %v", err)
+			}
```

Comment:
we need a `return` here when `err == nil`, otherwise we will always return error online 238. Is this code path never exercised?